com.unity.shadergraph/Editor/GraphDeltaRegistry/Registry/DefaultRegistry.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Runtime.Serialization;
+using System.Collections.Generic;
+using UnityEditor.ShaderGraph.GraphDelta;
+using System.Linq;
+using UnityEditor.ShaderGraph.Registry.Defs;
+using UnityEditor.ShaderGraph.Registry.Types;
+using com.unity.shadergraph.defs;
+
+namespace UnityEditor.ShaderGraph.Registry.Default
+{
+    public class DefaultContext : IContextDescriptor
+    {
+        public IReadOnlyCollection<IContextDescriptor.ContextEntry> GetEntries()
+        {
+            return new List<IContextDescriptor.ContextEntry>()
+            {
+                new ()
+                {
+                    fieldName = "BaseColor",
+                    primitive = GraphType.Primitive.Float,
+                    precision = GraphType.Precision.Fixed,
+                    height = GraphType.Height.One,
+                    length = GraphType.Length.One,
```

Comment:
Oh weird-- this is a Length 1 on the output-- I'm not surprised at all "it just works."